tests/Custom/TimeoutTest.php

```diff
@@ -64,4 +65,17 @@ public function testLongTimeoutWaitsForElementToAppear(): void
 
         $this->assertNotNull($element);
     }
+
+    public function testShortPageLoadTimeoutThrowsException(): void
+    {
+        $session = $this->getSession();
+        $driver = $session->getDriver();
+        \assert($driver instanceof WebdriverClassicDriver);
+
+        $driver->setTimeouts(array('page' => 1)); // Use 1ms timeout to avoid any successful page load.
+
+        $this->expectException('\Behat\Mink\Exception\DriverException');
```

Comment:
Please use class reference (`DriverException::class`)...string class names are often times forgotten and/or not properly recognized by static code analysers.